theano/gpuarray/opt.py

```diff
@@ -2182,6 +2247,11 @@ def local_gpu_svd(op, context_name, inputs, outputs):
 # to avoid a circular dependency problem with dnn
 from .dnn import (local_abstractconv_cudnn, local_abstractconv_gw_cudnn,
                   local_abstractconv_gi_cudnn)     # noqa: 402
+
+conv_metaopt = ConvMetaOptimizer()
+abstractconv_groupopt.register('conv_metaopt', conv_metaopt, 0, 'conv_meta')
```

Comment:
If I register the metaoptimizer before the registering others, it works. But If I register at the end with the integer argument as 0 or even 10. It does not work. 
But if i explicitly add position=10(or anything lesser) to the register function after registering everything else, it works